mongo/bulk_write_models.go

```diff
@@ -70,7 +70,8 @@ func (dom *DeleteOneModel) SetCollation(collation *options.Collation) *DeleteOne
 // specification as a document. This option is only valid for MongoDB versions >= 4.4. Server versions >= 3.4 will
 // return an error if this option is specified. For server versions < 3.4, the driver will return a client-side error if
 // this option is specified. The driver will return an error if this option is specified during an unacknowledged write
-// operation. The default value is nil, which means that no hint will be sent.
+// operation. The driver will return an error if the hint parameter is a multi-key map. The default value is nil, which
+// means that no hint will be sent.
```

Comment:
I see that `IndexModel.Keys` already has a note. Can you add the note to `Sort` options as well?

Reply:
done